examples/expositional/frameworks/cortexchat/cortex_chat_quickstart.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cortex Chat + TruLens\n",
+    "\n",
+    "This quickstart assumes you already have a Cortex Search Service started, JWT token created and Cortex Chat Private Preview enabled for your account. If you need assistance getting started with Cortex Chat, please contact your Snowflake contact."
```

Comment:
any public resources we can point people towards for creating a token or enabling/checking enablement of chat API PrPr?

Reply:
Added resources for cortex search and creating a JWT token. Common practice for PrPr features is to point to account team, as this should be a highly controlled process.